mpf/devices/multiball_lock.py

```diff
@@ -305,3 +305,26 @@ def _post_events(self, device, **kwargs):
     def _request_new_balls(self, balls):
         """Request new ball to playfield."""
         self.source_playfield.add_ball(balls=balls)
+    
+    def compensate_lock(self):
+        # evilmopey solution for issue 1278
+        if self.config['locked_ball_counting_strategy'] == "no_virtual":
+            self._locked_balls = 0
+            # remove balls that were in lock
+            balance = (self._locked_balls - (-1 * self._physically_locked_balls))  # -1 to make the number positive
+            for i in range(1, balance):
```

Comment:
I don't think we need to remove balls. That would give the current player more balls.